types/office-js-preview/index.d.ts

```diff
@@ -12831,6 +12902,44 @@ declare namespace Office {
          * @param input - The local time value to convert.
          */
         convertToUtcClientTime(input: LocalClientTime): Date;
+        /**
+         * Disables the Outlook Native Signature.
+         *  
+         * For Windows and Mac rich clients, this API sets the signature under the "New Message" and "Forward" sections
```

Comment:
For Windows and Mac rich clients, this API sets the default signature under the "New Message" and "Replies/Forwards" sections

Reply:
Updated